config/locales/en.yml

```diff
@@ -77,13 +80,15 @@ en:
     non_adwords: "Non adwords"
     non_adword_links: "Non adword links"
     not_found: "Keyword not found"
+    not_found_with_id: "Couldn't find Keyword with 'id'=%{id}"
     search: "Search"
     search_keywords: "Search keywords"
     status: "Status"
     top_pos_adwords: "Top position adwords"
     top_pos_adword_links: "Top position adword links"
     upload: "Upload"
     upload_csv_successfully: "CSV uploaded successfully"
+    upload_csv_unsuccessfully: "CSV uploaded unsuccessfully"
```

Comment:
Save as the above comment.

Reply:
Fixed in https://github.com/gutakk/ruby-google-scraper/pull/17 as I also don't use this anymore